notebook/notebook/notebook.py

```diff
@@ -131,10 +171,153 @@ def external_url_for(path):
     return url + path
 
 
+def forbidden():
+    return 'Forbidden', 403
+
+
 @app.route('/healthcheck')
 def healthcheck():
     return '', 200
 
+##### New notebook methods meant for consumption from agnostic client#####
+###### Methods that do not call the auth-gateway /verify method must not be directly public #######
```

Comment:
no comments

Reply:
I disagree with this. We need description of inappropriate use, for something that must occur outside of the context of the present package (i.e /auth requests). It is not at all clear that having a "User" header present in the request means that the request is secure.